gradle-typescript/src/main/java/com/gradlets/gradle/typescript/JestTestSupport.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * (c) Copyright 2021 Felipe Orozco, Robert Kruszewski. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.gradlets.gradle.typescript;
+
+import org.gradle.api.Project;
+
+public final class JestTestSupport {
+
+    private JestTestSupport() {}
+
+    static void addDependencyConstraints(Project project, SourceSet testSourceSet) {
+        project.getConfigurations()
+                .named(testSourceSet.getCompileConfigurationName())
+                .configure(conf -> {
+                    conf.getDependencyConstraints()
+                            .add(project.getDependencies().getConstraints().create("npm:types/node:14.6.2"));
```

Comment:
can we pull out all of the coordinates into a set and save on the boilerplate?

Reply:
we can